plugins/transport-reactor-netty4/src/main/java/org/opensearch/http/reactor/netty4/ReactorNetty4HttpRequest.java

```diff
@@ -156,6 +156,8 @@ public HttpVersion protocolVersion() {
             return HttpRequest.HttpVersion.HTTP_1_0;
         } else if (protocol.equals(io.netty.handler.codec.http.HttpVersion.HTTP_1_1.toString())) {
             return HttpRequest.HttpVersion.HTTP_1_1;
+        } else if (protocol.equals("HTTP/2.0")) {
```

Comment:
Very nitpicky nitpick: Add a `static HttpVersion parse(String s) {...}` to remove this logic from callers and keep it by the enum where values are added.

Reply:
Thanks @peternied , those are 2 different modules, but moving it to `HttpRequest` (as `static HttpVersion parse(String s) {...}`) would not let `io.netty.handler.codec.http.HttpVersion` to be used (since there is no netty dependencies in core).